test/Tests.Infrastructure/ConnectionString/MssqlConnectionString.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Data.SqlClient;
+
+namespace Tests.Infrastructure.ConnectionString
+{
+    public class MssqlConnectionString: SqlConnectionString<SqlConnection>
+    {
+        public static readonly MssqlConnectionString Instance = new MssqlConnectionString();
+
+        private MssqlConnectionString(): base("RAVEN_MSSQL_CONNECTION_STRING")
+        {
+        }
+
+        protected override string VerifiedConnectionStringFactor()
```

Comment:
Maybe the overriding is unnecessary? If we want to use just the mssql from the remote VM?